src/cascadia/TerminalApp/ActionAndArgs.h

```diff
@@ -7,11 +7,13 @@ namespace winrt::TerminalApp::implementation
 {
     struct ActionAndArgs : public ActionAndArgsT<ActionAndArgs>
     {
-        static const std::map<std::string_view, ShortcutAction, std::less<>> ActionNamesMap;
+        static const std::map<std::string_view, ShortcutAction, std::less<>> ActionKeyNamesMap;
```

Comment:
now should _this_ become a `std::unordered_map`?

Reply:
If I do the "sort alphabetically entries with the same weight" thing, then _yes_. If I just do the `unordered_map` without that, they're in _super random_ order.